zerver/views/webhooks/taiga.py

```diff
@@ -0,0 +1,231 @@
+            'changed_us': u':clipboard: %(user)s moved task **%(subject)s** from user story %(old)s to %(new)s.'
+        },
+        'issue': {
+            'create': u':bulb: %(user)s created issue **%(subject)s**.',
+            'set_assigned_to': u':busts_in_silhouette: %(user)s assigned issue **%(subject)s** to %(new)s.', #
+            'changed_assigned_to': u':busts_in_silhouette: %(user)s reassigned issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_priority': u':rocket: %(user)s changed priority of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_severity': u':warning: %(user)s changed severity of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_status': u':chart_with_upwards_trend: %(user)s changed status of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_type': u':bulb: %(user)s changed type of issue **%(subject)s** from %(old)s to %(new)s.',
+            'renamed': u':notebook: %(user)s renamed issue %(old)s to **%(new)s**.',
+            'description': u':notebook: %(user)s updated description of issue **%(subject)s**.',
+            'commented': u':thought_balloon: %(user)s commented on issue **%(subject)s**.',
+            'delete': u':x: %(user)s deleted issue **%(subject)s**.'
+        },
+    }
+
+
+    def get_old_and_new_values(change_type, message):
+        """ Parses the payload and finds previous and current value of change_type."""
+        values_map = {
+            'assigned_to': 'users',
+            'status': 'status',
+            'severity': 'severity',
+            'priority': 'priority',
+            'milestone': 'milestone',
+            'type': 'type',
+            'user_story': 'user_story'
+        }
+
+        if change_type in ['subject', 'name', 'estimated_finish', 'estimated_start']:
+            old = message["change"]["diff"][change_type]["from"]
+            new = message["change"]["diff"][change_type]["to"]
+            return old, new
+
+        try:
+            old_id = message["change"]["diff"][change_type]["from"]
+            old = message["change"]["values"][values_map[change_type]][str(old_id)]
+        except KeyError:
+            old = None
+
+        try:
+            new_id = message["change"]["diff"][change_type]["to"]
+            new = message["change"]["values"][values_map[change_type]][str(new_id)]
+        except KeyError:
+            new = None
+
+        return old, new
+
+
+    def parse_comment(message):
+        """ Parses the comment to issue, task or US. """
+        return {
+            'event': 'commented',
+            'type': message["type"],
+            'values': {
+                'user': message["change"]["user"]["name"],
+                'subject': message["data"]["subject"] if "subject" in list(message["data"].keys()) else message["data"]["name"]
+            }
+        }
+
+    def parse_create_or_delete(message):
```

Comment:
I noticed that you are using here the same structure of dictionary. Maybe it will be more clear way to simply use classes?